osc/_private/store.py

```diff
@@ -0,0 +1,316 @@
+"""
+Store class wraps access to files in the '.osc' directory.
+It is meant to be used as an implementation detail of Project and Package classes
+and shouldn't be used in any code outside osc.
+"""
+
+
+import os
+
+from . import api
+from .. import core as osc_core
+from .. import oscerr
+
+
+class Store:
+    STORE_DIR = ".osc"
+    STORE_VERSION = "1.0"
+
+    @classmethod
+    def resolve_project_name(cls, project_name):
+        """
+        Resolve special project names:
+            "." -> project name from the current working copy of a project or a package
+        """
+        if project_name == "/":
+            return None
+        elif project_name == ".":
+            store = cls(os.getcwd())
+            return store.project
+        return project_name
+
+    @classmethod
+    def resolve_package_name(cls, package_name):
+        """
+        Resolve special package names:
+            "." -> package name from the current working copy of a package
+        """
+        if package_name == ".":
+            store = cls(os.getcwd())
```

Comment:
Use `Path.cwd()|

Reply:
I have decided to remove these classmethods completely and bring them back when they'll be used.
I'm not entirely sure how they should look like.